views/wsprov_fetch_linked_objects.aql

```diff
@@ -8,10 +8,9 @@
 //   sublink_limit - number of sublink results (10-20 recommended)
 
 for o in wsprov_object
-  filter o._key IN @obj_keys
-  filter o.is_public || (o.workspace_id IN @ws_ids)
-  for link in 1..100 any o wsprov_links
+  filter o._key in @obj_keys
+  for link in 1..10 any o wsprov_links
```

Comment:
If I understand correctly, this effectively sets the max depth to follow links to to 10? Is this  `@sublink_limit`? Do you want to use the variable here or remove the comment on line 8?

Reply:
10 is the max traversal depth. So you'd at most get links of links of links of links of links (up to 10 levels). I added this to try and reduce the query time
"sublink limit" i actually need to remove in the comments. The previous UI loaded links plus sublinks in one request. The newer UI will just load object links in one flat list. Then, if you click a link, it makes a separate request for those sublinks.

Reply:
👍 Thx